### DISCUSSION TITLE
Welcome to geOrchestra Discussions!

### DISCUSSION THREAD

**Discussion by fvanderbiest** [georchestra/georchestra#3217] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ChiranthTL** [comment]
Hey !! would need help with something on geonetwork which is part of georchestra...is this the right place to ask ?

**Reply by fvanderbiest** [reply]
Here is fine too :-)

**Comment by pierrejego** [comment]
Hi,
better place would be threre : https://groups.google.com/g/georchestra or https://groups.google.com/g/georchestra-dev
